ta_services/keymaster/include/keymaster_ta.h

```diff
@@ -15,6 +17,6 @@
  *
  * in	params[0].memref: entropy input data
  */
-#define KEYMASTER_CMD_ADD_RNG_ENTROPY	0
+#define KEYMASTER_CMD_ADD_RNG_ENTROPY	1
```

Comment:
Don't change this. Before we decide to merge everything to master we can renumber and change order of stuff etc.

Reply:
Ok. Will change back.